addons/delivery/models/delivery_zip_prefix.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class DeliveryZipPrefix(models.Model):
+    """ Zip prefix that a delivery.carrier will deliver to. """
+    _name = 'delivery.zip.prefix'
+    _description = 'Delivery Zip Prefix'
+
+    name = fields.Char('Prefix', required=True, translate=True)
```

Comment:
Are your sure the name needs to be translated?
As I see it, a zip prefix is static, independently of the language :thinking: